### PR TITLE
Deploying and scaling ASP.NET Core on Azure Container Apps

### PR DESCRIPTION
**Pull request by alexwolfmsft**
## Summary

Describe your changes here.

Fixes #Issue_Number (if available)

### CODE REVIEW

**Comment by alexwolfmsft** [comment]
This draft is ready for general review and feedback. I'm still making verbiage changes and polishing it up, but any feedback on the content and structure of the article would be great. For some reason screenshots are loading intermittently so you may have to refresh a few times to see them.

**Comment by alexwolfmsft** [comment]
Adding @bradygaster  @Rick-Anderson

**Comment by Rick-Anderson** [comment]
@alexwolfmsft why is this in .NET and not in ASP.NET Core?
Take a look at the build report.

**Comment by Rick-Anderson** [comment]
@alexwolfmsft  I recommend you close this PR and we continue working on it at https://github.com/dotnet/AspNetCore.Docs/pull/26869
cc @bradygaster

File: docs/azure/sdk/scaling-aspnetcore-apps.md [hunk 1]
```diff
@@ -0,0 +1,352 @@
...
+    ---
+
+1. Update the `Program.cs` file to include the following `using` statements and code:
+
+    ```csharp
+    using Azure.Identity;
+    using Microsoft.AspNetCore.DataProtection;
+    using Microsoft.Extensions.Azure;
+    
+    builder.Services.AddAzureClientsCore();
```
Comment (scottaddie) [inline docs/azure/sdk/scaling-aspnetcore-apps.md:269]: What's the motivation for including this line? Does something not work properly if it's excluded? If your goal is to enable forwarding of Azure SDK for .NET logs, use the approach documented here instead: https://docs.microsoft.com/dotnet/azure/sdk/logging#logging-without-client-registration.

File: docs/azure/sdk/scaling-aspnetcore-apps.md [hunk 2]
```diff
@@ -0,0 +1,352 @@
...
+    * **Azure.Identity**: Provides classes to work with the Azure identity and access management services.
+    * **Microsoft.AspNetCore.DataProtection**: Provides services to configure data protection.
+    * **Microsoft.Extensions.Azure**: Provides helpful extension methods to perform core Azure configurations.
+
+    ## [.NET CLI](#tab/cli)
+
+    ```dotnetcli
+    dotnet add package Azure.Identity
+    dotnet add package Azure.Extensions.AspNetCore.DataProtection.Blobs
+    dotnet add package Azure.Extensions.AspNetCore.DataProtection.Keys
```
Comment (scottaddie) [inline docs/azure/sdk/scaling-aspnetcore-apps.md:253]:
```suggestion
    dotnet add package Azure.Extensions.AspNetCore.DataProtection.Keys
    dotnet add package Microsoft.Extensions.Azure
```

**Pull request closed without merging**